src/commands/cloud-run/flare.ts

```diff
@@ -11,19 +11,30 @@ import chalk from 'chalk'
 import {Command} from 'clipanion'
 import {GoogleAuth} from 'google-auth-library'
 
-import {API_KEY_ENV_VAR, CI_API_KEY_ENV_VAR, FLARE_OUTPUT_DIRECTORY, LOGS_DIRECTORY} from '../../constants'
-import {sendToDatadog} from '../../helpers/flare'
+import {
+  ADDITIONAL_FILES_DIRECTORY,
+  API_KEY_ENV_VAR,
+  CI_API_KEY_ENV_VAR,
+  FLARE_OUTPUT_DIRECTORY,
+  FLARE_PROJECT_FILES,
```

Comment:
Cool, I wonder if in the future, the flare constant should be inside the `helpers/flare.ts` file? nit, not a blocker

Reply:
Good question. I could actually move all the flare constants to `helpers/flare.ts`, or we could keep them here in `src/constants.ts`

Currently, these constants are only used by the `cloud-run` and `lambda` commands, so they are kind of flooding the constants file. But eventually, they will also be used by the Azure `app-service` command.

@Drarig29 do you think they should be moved?

Reply:
I think for now, the `src/constants.ts` is a good place. As long as it's well organized, I don't think this file being long is an issue. And IIUC, those flare constants will soon be shared by 3 commands.

Maybe in the future we can split `src/constants.ts` into multiple files, but I don't think it's needed right now. Let alone in this PR.

Reply:
@nhulston I would rather keep constants in `src/constants.ts`, even if they are specific to sending a flare, which can be achieved my multiple commands.